examples/getting_started/3_Deep_dive_into_the_anatomy_of_quantum_circuits/3_Deep_dive_into_the_anatomy_of_quantum_circuits.ipynb

```diff
@@ -100,17 +100,20 @@
      "name": "stdout",
```

Comment:
<p>We can see that the <code>measure</code> instruction, represented by <code>M</code> in the diagram, was added to our circuit, and only on qubit 0.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/amazon-braket/amazon-braket-examples/pull/540/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='55'/>